### PR TITLE
Polish onboarding documentation

### PR DESCRIPTION
**Pull request by grahamalama**
As a follow-up of https://github.com/mozilla/jira-bugzilla-integration/pull/738, add some more polish to our onboarding documentation

- include the specific permissions that are needed for the Jira Automation bot
- mention the issue template for non-technical users who want to set up a project sync
- tweak the "architecture" of the steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,19 +43,32 @@ graph TD
...
+     See [actions documentation](docs/actions.md)
+   - If not, submit an issue [here](https://github.com/mozilla/jira-bugzilla-integration/issues/new?assignees=&labels=configuration&projects=&template=new-config.yaml&title=Sync+%3CBugzilla+Product%3E+with+%3CJira+Project%3E), and we'll set up your configuration
+1. Grant permissions to the Jira Automation Bot
+
+   - If you are an admin of the Jira project
+
+     - go to your Jira project and open `Project Settings`, then `People`.
+     - Select `Add People` and search for `Jira Automation`. If two are listed select the one with the green logo
+     - From the `Roles` drop down select `Bots`. Click `Add 1 person`.
+     - Add these permission for the bot
```
Comment (leplatrem) [inline README.md:57]:
```suggestion
     - Add these permissions for the bot
```

File: README.md [hunk 2]
```diff
@@ -43,19 +43,32 @@ graph TD
...
+
+     ```
+        "ADD_COMMENTS",
+        "CREATE_ISSUES",
+        "DELETE_ISSUES",
+        "EDIT_ISSUES"
+     ```
+
+   - If you are not an admin of the Jira project contact the admin to determine how best to request the changes described above
```
Comment (leplatrem) [inline README.md:66]: Shall we mention `#jira-support` here?

File: README.md [hunk 3]
```diff
@@ -43,19 +43,32 @@ graph TD
...
+
+1. Once your configuration is merged and a release is created, create a bug in Bugzilla and add your whiteboard tag to the bug. Note that the tag must be enclosed in square brackets (eg. `[famous-project]`). You should see an issue appear in Jira
```
Comment (leplatrem) [inline README.md:68]:
```suggestion
1. Once your configuration is merged and a JBI release is deployed, create a bug in Bugzilla and add your whiteboard tag to the bug. Note that the tag must be enclosed in square brackets (eg. `[famous-project]`). You should see an issue appear in Jira
```

File: README.md [hunk 4]
```diff
@@ -43,19 +43,32 @@ graph TD
...
+   - If you want to start syncing a bug to a Jira issue that already exists, add the issue's link to the `See Also` section of the bug before you add the whiteboard tag
```
Comment (leplatrem) [inline README.md:69]:
```suggestion
   - If you want to start syncing a bug to a Jira issue that already exists, add the issue's link to the `See Also` section of the Bugzilla bug before you add the whiteboard tag
```

**Review by leplatrem — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (grahamalama) [reply README.md]: [4370b48](https://github.com/mozilla/jira-bugzilla-integration/pull/739/commits/4370b48b7e830bda0b3cd015ab467ef72f1ab8be)

**Comment by grahamalama** [comment]
Going to bypass branch requirements because that's unrelated to the changes in this PR. Going to open a PR to fix that shortly

**Pull request merged**